File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    s = s.lower()
    p_count, y_count = 0,0
    
    for i in s:
        if i == 'p':
            p_count +=1
        elif i == 'y':
            y_count +=1
    if p_count == y_count or (p_count == 0 and y_count == 0):
        return True
    else:
        return False
    
    

    return True

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
import java.util.*;

class Solution {
    boolean solution(String s) {
        boolean answer = true;
        
        int cnt = 0;
        s = s.toLowerCase();
        for (char c : s.toCharArray()) {
            if (c == 'p') 
                cnt++;
            else if (c == 'y')
                cnt--;
        }        

        return (cnt == 0) ? answer : !answer;
    }
}